### PR TITLE
Add procedure to create a mailling list

### PR DESCRIPTION
**Pull request by mscherer**

### CODE REVIEW

File: procedures/adding_a_list.rst [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+
+For now, the list are still managed in salt. This is expected to change
+in the future, and due to that, the process is not as streamlined
+as it could be.
+
+Step 1: Adding the list to the aliases file
+===========================================
+
+In order to add the alias, you need to modify the file mailing_lists.sls
+in the salt public pillar repository. Report to the documentation of salt for
```
Comment (nigelbabu) [inline procedures/adding_a_list.rst]: s/Report/Refer

File: procedures/adding_a_list.rst [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+    /usr/lib/mailman/bin/list_lists
+
+If the list appear, then all is good. If not, you need to create it manually with this command::
+
+    /usr/lib/mailman/bin/newlist NAME ADMIN NAME
+
+By convention, ADMIN should be the list-admin alias for gluster.org, and the password should be the same
+as the name of the list.
+
+All people in the list-admin alias will get the notification (in practice, tht mean people who are on the root
```
Comment (nigelbabu) [inline procedures/adding_a_list.rst]: *that means

File: procedures/adding_a_list.rst [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+alias), so you can forward it to the user who requested the list.
+At minima, you should make sure they change the password. They do not need to give it to anyone since
```
Comment (nigelbabu) [inline procedures/adding_a_list.rst:38]: Can we instead change the process to: As soon as the mailing list is created, reset the password so it's a random string. Forward this email to the requester.

**Comment by mscherer** [comment]
We can, but then we can as well set a ranom string in the first place for newlist).

**Pull request merged**